runhouse/resources/hardware/cluster.py

```diff
@@ -1128,6 +1129,41 @@ def _start_or_restart_helper(
         # Update the cluster config on the cluster
         self.save_config_to_cluster()
 
+        # Save a limited version of the local ~/.rh config to the cluster with the user's
+        # if such does not exist on the cluster
+        if rns_client.token:
+            user_config = {
+                "token": rns_client.cluster_token(resource_address=rns_client.username),
+                "username": rns_client.username,
+                "default_folder": rns_client.default_folder,
+            }
+
+            condensed_config_yaml_path = Path(
```

Comment:
maybe easier if we throw this in a temp dir, and then just nuke the temp dir after?

Reply:
is it basically what we are doing, no? I'm creating the tmp dir inside `.rh` dir, and call it `condensed_config` instead of just `temp`

Since we are saving the token in the condensed config.yaml, I don't want to save the file outside of the .rh local folder.

Reply:
@jlewitt1 yeah makes sense, changed it